website/pages/docs/internals/security.mdx

```diff
@@ -0,0 +1,322 @@
+    Containers](https://katacontainers.io/). These types of runtimes provide
+    sandboxing features which help prevent raw access to the underlying shared
+    kernel for other containers and the Nomad client agent itself.
+
+* **[Disable Unused Drivers](/docs/configuration/client#driver-blacklist)** -
+    Each driver provides different degrees of isolation, and bugs may allow
+    unintended privilege escalation. If a task driver is not needed, you can
+    disable it to reduce risk.
+
+* **Linux Security Modules** - Use of security modules that can be directly
+    integrated into operating systems such as AppArmor, SElinux, and Seccomp on
+    both the Nomad hosts and applied to containers for an extra layer of
+    security. Seccomp profiles are able to be passed directly to containers
+    using the
+    **[security_opt](/docs/drivers/docker.html#security_opt)**
+    parameter available in the default [Docker
+    driver](/docs/drivers/docker.html).
+
+* **[Service Mesh](https://www.hashicorp.com/resources/service-mesh-microservices-networking)** -
+    Integrating service mesh technologies such as
+    **[Consul](https://www.consul.io/)** can be extremely useful for limiting
+    and efficiently load balancing network connectivity within a cluster.
+
+### Threat Model
+
+The following are parts of the Nomad threat model:
+
+* **Nomad agent-to-agent communication** - Transport encryption for
+    agent-to-agent communication is required to prevent eavesdropping. TCP and
+    UDP based protocols within Nomad provide different mechanisms for enabling
+    encryption including symmetric (shared gossip encryption keys) and
+    asymmetric keys (TLS).
+
+* **Tampering of data in transit** - Any tampering should be detectable via mTLS
+    and cause Nomad to avoid processing the request.
+
+* **Access to data without authentication or authorization** - Requests to the
+    server should be authenticated and authorized using mTLS and ACLs
+    respectively.
+
+* **State modification or corruption due to malicious messages** - Improperly
+    formatted messages are discarded while properly formatted messages require
+    authentication and authorization.
+
+* **Non-server members accessing raw data** - All servers that join the cluster
+    require proper authentication and authorization in order to begin
+    participating in Raft. All data in Raft should be encrypted with TLS.
+
+* **Denial of Service against a node** - DoS attacks against a single node
+    should not compromise the security posture of Nomad.
+
+The following are not part of the threat model for server agents:
+
+* **Access (read or write) to the Nomad data directory** - Information about the
+    jobs managed by Nomad is persisted to a server’s data directory.
+
+* **Access (read or write) to the Nomad configuration directory** - Access to
+    Nomad’s configuration file(s) directory can enable and disable features for
```

Comment:
```suggestion
    Nomad's configuration file(s) directory can enable and disable features for
```

Also worth noting that on server agents this can expose highly-privileged Vault and Consul tokens.